cfme/configure/configuration/__init__.py

```diff
@@ -515,6 +515,47 @@ def update(self):
         self.appliance.server_details_changed()
 
 
+class VMwareConsoleSupport(Updateable, Pretty, Navigatable):
+    """
+    This class represents the "VMware Console Support" section of the Configuration page.
+    Note this is to support CFME 5.8 and beyond functionality.
+
+    Args:
+        console_type: VMware VMRC Plugin, VNC or VMware WebMKS
+
+    Usage:
+
+        vmware_console_support = VMwareConsoleSupport(console_type="VNC")
+        vmware_console_support.update()
+
+    """
+    vmware_console_form = Form(
```

Comment:
We discussed the possibility of instead implementing this as widgets, in a nested view in base.ui's definition for this part of the configuration form. Not necessary for this PR as this implementation matches the rest of the module.